### ISSUE TITLE
-BWSZ by itself does not plot the z-axis

### ISSUE THREAD

**Issue by joa-quim** [GenericMappingTools/gmt#295]
Doesn't plot z-axis
```
psbasemap  -R0/1/0/1/0/1 -JX10 -BWSZ1 -Bz+lZ -Bx+lX -By+lY -JZ10 -p45/30  -P > lixo.ps
```

One needs to cheat a bit, change ``-Bz`` to ``Bza2``, to make the z-axis show up but without annotations
```
psbasemap  -R0/1/0/1/0/1 -JX10 -BWSZ1 -Bza2+lZ -Bx+lX -By+lY -JZ10 -p45/30  -P > lixo.ps
```

**Comment by joa-quim** [comment]
Annotating every 2 when the length is 1 is the trick that I mention to make it draw the axis without annotations. 
 
(Suffering in Ikea so don’t ask more details)

Sent from my iDedo

No dia 07/01/2019, às 18:46, Paul Wessel <notifications@github.com> escreveu:

**Comment by joa-quim** [comment]
Fixed by #299
